sort-member-config.js

```diff
@@ -1,16 +1,17 @@
 module.exports.sortMemberRules = {
 	"sort-class-members/sort-class-members": [2, {
 		"order": [
-		  "[static-properties-getter]",
-		  "[static-styles-getter]",
-		  "[static-properties]",
-		  "[static-methods]",
-		  "[properties]",
-		  "constructor",
-		  "[accessor-pairs]",
-		  "[methods]",
-		  "[conventional-private-properties]",
-		  "[conventional-private-methods]",
+			"[static-property-is]",
```

Comment:
Adds a rule for "is" static property before the static "properties".  Also, I renamed the custom group names to be `static-property-`.